tools/crm_resource_runtime.c

```diff
@@ -1808,7 +1824,7 @@ cli_resource_execute_from_params(const char *rsc_name, const char *rsc_class,
             free(local_copy);
         }
     } else {
-        rc = op->rc == 0 ? pcmk_err_generic : op->rc;
+        rc = op->rc == 0 ? pcmk_rc_error: op->rc;
```

Comment:
This is where it would use CRM_EX_ERROR (op->rc is actually the exit status of the subprocess spawned for the op)